src/category_theory/isomorphism.lean

```diff
@@ -205,6 +205,18 @@ is_iso.of_iso $ (as_iso f) ≪≫ (as_iso h)
 @[simp] lemma iso.inv_inv (f : X ≅ Y) : inv (f.inv) = f.hom := rfl
 @[simp] lemma iso.inv_hom (f : X ≅ Y) : inv (f.hom) = f.inv := rfl
 
+lemma eq_is_iso_comp (α : X ⟶ Y) [is_iso α] {f : X ⟶ Z} {g : Y ⟶ Z} : f = α ≫ g ↔ inv α ≫ f = g :=
```

Comment:
Does it make sense to flip the sides of all these four lemmas, and mark them `simp`, so that you can simp away the `inv`s?

Reply:
Done.